internal/delivery/http/handlers/feedbacks.go

```diff
@@ -0,0 +1,209 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/go-chi/chi/v5"
+	"github.com/google/uuid"
+
+	"github.com/andrsj/feedback-service/internal/domain/models"
+)
+
+const (
+	defaultLimit    = 10
+	limitQueryParam = "limit"
+	nextQueryParam  = "next"
+)
+
+var (
+	errIDParamIsMissing = errors.New("id parameter is missing")
+	errLimitParam       = errors.New("invalid limit parameter")
+	errNextParam        = errors.New("invalid next parameter")
+	errNoValuesNext     = errors.New("no values after 'next'")
+)
+
+// GetFeedback GET /feedback/{id}.
+func (h *Handlers) GetFeedback(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+
+	feedbackID := chi.URLParam(r, "id")
+	if feedbackID == "" {
+		h.handleError(w, http.StatusBadRequest, errIDParamIsMissing)
+
+		return
+	}
+
+	feedback, err := h.feedbackService.GetByID(feedbackID)
```

Comment:
Usually, for working with databases we use context https://www.alexedwards.net/blog/how-to-manage-database-timeouts-and-cancellations-in-go

How about passing `r.Context()`?
```suggestion
	feedback, err := h.feedbackService.GetByID(r.Context(), feedbackID)
```

Of course, we should change also `GetByID` implementation (and other functions).

Reply:
I would say not "Usually", but always.

Reply:
In some DBs you can set timeout instead of using Context, for example in RethinkDB it's not required, if you set all necessary timeouts according to your requirements.

Reply:
@driif, it's more about the circuit breaker - it makes no sense to burn the CPU of DB if the HTTP client has disconnected.